src/core/iTextSharp/text/rtf/parser/destinations/RtfDestinationShppict.cs

```diff
@@ -74,7 +74,7 @@ public class RtfDestinationShppict : RtfDestination {
         private StringBuilder buffer = new StringBuilder();
 
         /* picttype */
-        private int pictureType = Image.ORIGINAL_NONE;
+      //  private int pictureType = Image.ORIGINAL_NONE;

```

Comment:
In order to remain true to the original library, this should be remain uncommented.